Khanm last 3

ilikerustoo opened the pull request:
This is my part of the work

Commit:
This is my first commit

Commit:
My part completed

gkapfham commented:
Hello @ilikerustoo, I wanted to let you know that we cannot merge this pull request because of the fact that is is currently breaking the build. I should also point out that "This is my part of the work" does not describe the work &emdash; without a complete description of your contributions we will not merge this pull request. Finally, please note that none of your commit messages actually explain how your commits changed the code base. You should also notice that you did not request a review for this PR! Also, @ilikerustoo, please make sure that you add some type of identifying icon or photograph to your GitHub profile!

Commit:
Fully completed the last 3 questions within the 'Software Development with Python' subsection. Fixed mdl markdown issues to pass Travis CI check

ilikerustoo commented:
Thank you for the feedback. I meant to add a pull request to our fork but did here accidentally but I fixed the CI check and added a descriptive commit. I have also added a GitHub profile picture but it looks it may take a while for it to update everywhere.
